Add ignored patches feature to PaStA

8sd opened the pull request:
This pull request adds a feature to detect ignored patches.
In addition, the pull request contains a feature to export the characteristics of the patches analyzed.

8sd committed:
Add script to evaluate patches

Signed-off-by: Sebastian Duda <sebastian.duda@fau.de>

rralf committed:
evaluate_patches: Add R script for evaluation of characteristics.csv

Add the evaluation script.

Signed-off-by: Ralf Ramsauer <ralf.ramsauer@oth-regensburg.de>

rralf committed:
evaluate_patches: Don't pass repo to get_ignored

It's not used.

Signed-off-by: Ralf Ramsauer <ralf.ramsauer@oth-regensburg.de>

rralf committed:
evaluate_patches: add some documentation

Signed-off-by: Ralf Ramsauer <ralf.ramsauer@oth-regensburg.de>

rralf reviewed (changes requested):
Could you please squash those changes to the patch "Add script to evaluate patches"?

rralf commented:
... And please rebase to latest next. Otherwise, things look fine. Thanks!

8sd committed:
evaluate_patches: implement business logic

Implement a script adds specifix properties to certain types of Linux patches.
Export those properties as csv file.

Signed-off-by: Sebastian Duda <sebastian.duda@fau.de>

8sd committed:
Merge remote-tracking branch 'upstream/next' into ignored_patches_1

8sd commented:
@rralf I did both tasks

rralf commented:
Thanks, applied.

rralf commented on bin/pasta_evaluate_patches.py:
```diff
@@ -2,13 +2,319 @@
 def evaluate_patches(config, prog, argv):
+<<<<<<< HEAD
     pass
+=======
```
Huh? How did that happen? Will remove this chunk and rewrite next.
